testing/integration/test.py

```diff
@@ -0,0 +1,45 @@
+import requests
+#example api test
+
+import json
+import sys
+sys.path.append('../../taketwo-webapi')
+sys.path.append('../util')
+from main import app
+from main import validate
+from main import getDb
+from fastapi.testclient import TestClient
+from util.assert_util import comparePayloads
+
+def override_validate():
+    return {"sub": 'test'}
+
+app.dependency_overrides[validate] = override_validate
+
+def test_save_mark():
+    url = '/mark'
+    
+    # Additional headers.
+    headers = {'Content-Type': 'application/json' } 
+
+    # Body
+    payload = {'user_id': "test", 'flagged_string': "test string", 'category': "stereotyping", 'info': "none", 'url': "example.com"}
+    
+    client = TestClient(app)
+    # convert dict to json by json.dumps() for body data. 
+    resp = client.post(url, headers=headers, data=json.dumps(payload,indent=4))       
+    
+    # Validate response headers and body contents, e.g. status code.
+    assert resp.status_code == 200, "Unexpected status code. Was: " + str(resp.status_code)
+    resp_body = resp.json()
+    assert resp_body['url'] == 'example.com', "Unexpected url. Was: " + resp_body['url']
+
+    #make sure payload stored properly
+    storedPayload = getDb().get(resp_body["_id"])
+    comparePayloads(payload, storedPayload)
```

Comment:
Different from the previous version, add comparison of what's stored in the db to the payload we send to make sure it's stored properly